lib/Twig/Loader/Filesystem.php

```diff
@@ -163,6 +163,8 @@ public function exists($name)
         try {
             return false !== $this->findTemplate($name, false);
         } catch (Twig_Error_Loader $exception) {
+            @trigger_error(sprintf('In %s::findTemplate(), you must accept a second argument that when set to "false" returns "false" instead of throwing an exception. Not supporting this argument is deprecated since version 1.27.', __CLASS__), E_USER_DEPRECATED);
```

Comment:
you should use ``get_class($this)`` instead of ``__CLASS__`` (or detect the declaring class of ``findTemplate`` to be even more precise).
Otherwise, the message says that the issue is in Twig_Loader_Filesystem while it is actually in a child class